src/fshtypes/Quantity.ts

```diff
@@ -0,0 +1,5 @@
+import { Code } from './Code';
+
+export class Quantity {
+  constructor(public value: number, public unit: Code) {}
```

Comment:
I failed to mention this (and perhaps even implied otherwise), but `unit` is not actually required -- so that should be `public unit?: Code`.

Reply:
It was such a trivial change, I just went ahead and made it.